pkg/config/derived-role-cert.go

```diff
@@ -46,7 +48,7 @@ func (idCfg *IdentityConfig) derivedRoleCertConfig() error {
 	// Enabled from now on:
 	idCfg.RoleCert = DerivedRoleCert{
 		Use:               true,
-		Dir:               idCfg.roleCertDir,
+		Dir:               strings.TrimSuffix(idCfg.roleCertDir, "/") + "/",
```

Comment:
```suggestion
		Dir:               strings.TrimSuffix(idCfg.roleCertDir, "/") + "/", // making sure it always ends with `/`
```

Reply:
fixed in: https://github.com/AthenZ/k8s-athenz-sia/pull/146/commits/99dc0513b9dccfeae06cc1bff816cbd8b5ddb2da